### PR TITLE
CSV import ad display functionality

### PR DESCRIPTION
**Pull request by adipundir**
Added csv file import functionality and display the contents of the csv file in a table.
the data is cleaned and arranged in an array of structs.

https://github.com/user-attachments/assets/c560fb6b-995d-4ed3-8fd0-2cbee8832410

### CODE REVIEW

File: public/token_transfer_info.csv [hunk 1]
```diff
@@ -0,0 +1,11 @@
+walletAddress,chain,amount,currency
+0x0123456789abcdef101112131415161718191a1b1c1d1e1f2021222324252627,Ethereum,0.0,ETH
```
Comment (bicced) [inline public/token_transfer_info.csv:2]: delete this file

**Review by bicced — COMMENTED** [review]
oh man.. just merge it

File: src/app/multisender/helper.ts [hunk 2]
```diff
@@ -0,0 +1,58 @@
+export type TokenTransferInfo = {
+    walletAddress : `0x${string}`
+    chain: string
+    amount: number 
+    currency: "ETH" | "USDC" | "USDT"
+}
+
+export const sampleTokenTransferInfo: TokenTransferInfo[] = [
```
Comment (adipundir) [inline src/app/multisender/helper.ts:8]: the sampleTokenTransferInfo is what is shown in the csv schema example.
without it it's just gonna be an empty table.

Screenshot 2024-08-29 at 1 22 16 AM

Comment (bicced) [reply src/app/multisender/helper.ts:8]: Lets remove this, and have a template download button (which provides a formatted csv with correct schema)

**Pull request closed without merging**